docs/installation/octopus-in-container/troubleshooting-octopus-server-in-a-container.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Troubleshooting Octopus Server in a Container
+description: An Octopus Deploy Server instance can be run directly from within a container.
+position: 10
+---
+
+## Accept the EULA
+
+When you create an Octopus Server container, you must agree with the [Octopus Deploy EULA](https://octopus.com/company/legal).
+
+If you get an error similar to:
+
+```
+ERROR: You must accept the EULA
+```
+
+You must pass `--env "ACCEPT_EULA=Y"` when using `docker run` with an Octopus Server image. By passing this value, you accept the Octopus Deploy EULA.
```

Comment:
```suggestion
You must pass `--env "ACCEPT_EULA=Y"` when using `docker run` with an Octopus Server image. By passing this value, you signify acceptance of the Octopus Deploy EULA.
```

Reply:
`signify acceptance of` feels quite wordy. We could go with:
```suggestion
You must pass `--env "ACCEPT_EULA=Y"` when using `docker run` with an Octopus Server image to accept the Octopus Deploy EULA.
```